source/Calamari.Shared/Log.cs

```diff
@@ -7,225 +7,206 @@
 
 namespace Calamari
 {
-    public class LogWrapper : ILog
-    {
-        public void Verbose(string message)
-        {
-            Log.Verbose(message);
-        }
-
-        public void VerboseFormat(string message, params object[] args)
-        {
-            Log.VerboseFormat(message, args);
-        }
-
-        public void Info(string message)
-        {
-            Log.Info(message);
-        }
-
-        public void InfoFormat(string message, params object[] args)
-        {
-            Log.Info(message, args);
-        }
-
-        public void Warn(string message)
-        {
-            Log.Warn(message);
-        }
-
-        public void WarnFormat(string message, params object[] args)
-        {
-            Log.WarnFormat(message, args);
-        }
-
-        public void Error(string message)
-        {
-            Log.Error(message);
-        }
-
-        public void ErrorFormat(string message, params object[] args)
-        {
-            Log.ErrorFormat(message, args);
-        }
-
-    }
-
     public interface ILog
     {
         void Verbose(string message);
-        void VerboseFormat(string message, params object[] args);
+        void VerboseFormat(string messageFormat, params object[] args);
         void Info(string message);
-        void InfoFormat(string message, params object[] args);
+        void InfoFormat(string messageFormat, params object[] args);
         void Warn(string message);
-        void WarnFormat(string message, params object[] args);
+        void WarnFormat(string messageFormat, params object[] args);
         void Error(string message);
-        void ErrorFormat(string message, params object[] args);
+        void ErrorFormat(string messageFormat, params object[] args);
+        void SetOutputVariableButDoNotAddToVariables(string name, string value, bool isSensitive = false);
```

Comment:
Renamed this overload to make it clear it doesn't add to the current variable dictionary. Looking at the callers most of them do anyway. Left for another day.